src/Android/Accessibility/AccessibilityHelpers.cs

```diff
@@ -115,6 +115,18 @@ public static class AccessibilityHelpers
             "com.ss.squarehome2",
             "com.treydev.pns"
         };
+        
+        // Be sure to keep these entries sorted alphabetically
+        public static Dictionary<string, KnownUsernameField> KnownUsernameFields => new List<KnownUsernameField>
+        {
+            new KnownUsernameField("accounts.google.com","ServiceLogin", "Email"),
+            new KnownUsernameField("amazon.com","signin", "ap_email_login"),
```

Comment:
Some formatting issues here with spacing.

Reply:
Sorry about that - thanks @cscharf for fixing